components/brave_wallet_ui/components/extension/connected-panel/index.tsx

```diff
@@ -99,7 +103,7 @@ const ConnectedPanel = (props: Props) => {
       />
       <CenterColumn>
         <StatusRow>
-          <OvalButton onClick={connectAction}>
+          <OvalButton disabled={activeOrigin === 'chrome://wallet/'} onClick={onShowSitePermissions}>
```

Comment:
Is there a possibility that the origin inside the wallet changes in future? Wondering if `.startsWith()` makes sense in that case.

Reply:
`activeOrigin` will always return the origin for an active tabs `url` so for instance
```
chrome://wallet/crypto/portfolio would return chrome://wallet/
```
So `startsWith()` would not make a difference in this case!